sites/example-project/src/pages/+layout.svelte

```diff
@@ -75,13 +75,92 @@
 		QueryStatus,
 		ToastWrapper
 	} from '@evidence-dev/core-components';
+
 	const prefetchStrategy = dev ? 'tap' : 'hover';
 
 	let mobileSidebarOpen = false;
 
 	$: if ($navigating) {
 		mobileSidebarOpen = false;
 	}
+
+	if (import.meta.hot) {
```

Comment:
This can likely be put into an `EvidenceErrorOverlay` component to remove it from the layout

Reply:
@ItsMeBrianD see: #1364